COPYRIGHT

```diff
@@ -6,16 +6,17 @@ terms.
 
 Longer version:
 
-The Rust Project is copyright 2010, The Rust Project
-Developers.
+Copyrights in the Rust project are retained by their contributors. No
+copyright assignment is required to contribute to the Rust project.
```

Comment:
If you include the "No copyright assignment is required" sentence, you should also talk about what *is* required, namely that by contributing, you agree to license the content under the given licenses and assert that you have the legal authority to license the content, or the content was given to you by a third party... or something like that.